lib/Cake/Network/Email/CakeEmail.php

```diff
@@ -1717,6 +1721,27 @@ protected function _renderTemplates($content) {
 			$rendered[$type] = implode("\n", $rendered[$type]);
 			$rendered[$type] = rtrim($rendered[$type], "\n");
 		}
+
+		/* Embed images inline in html templates */
+		if (!empty($rendered['html'])) {
```

Comment:
Of course there is a risk, in the first version a user could ask any file server without too much difficulty.
But now I determine accurately the files "user side" and the "template side" and included only the files of "template side".